src/main/java/com/adyen/model/checkout/CreateCheckoutSessionRequest.java

```diff
@@ -0,0 +1,1551 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2021 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+
+package com.adyen.model.checkout;
+
+import java.util.Objects;
+
+import com.adyen.model.AccountInfo;
+import com.adyen.model.Address;
+import com.adyen.model.Amount;
+import com.adyen.model.applicationinfo.ApplicationInfo;
+import com.adyen.model.Name;
+import com.adyen.model.Split;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
```

Comment:
```suggestion
import static com.adyen.util.Util.toIndentedString;
```
so you can remove the method at line 1542